src/libguac/guacamole/wol-constants.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef __GUAC_WOL_CONSTANTS_H
+#define __GUAC_WOL_CONSTANTS_H
```

Comment:
Though we (incorrectly) have used a double-underscore prefix for this in the past, doing so is technically not good and intrudes on a reserved namespace.

This should just be `GUAC_WOL_CONSTANTS_H`.

Reply:
ok

Reply:
Fixed.